ci-operator/step-registry/telcov10n/slcm/tests/telcov10n-slcm-tests-commands.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -e
+set -o pipefail
+
+CREDENTIALS_PATH=/var/run/stage-01
+CI_PROJECT_ID=`cat $CREDENTIALS_PATH/ci-project-id/CI_PROJECT_ID`
+DCI_REMOTE_CI=`cat $CREDENTIALS_PATH/dci-remote-ci/DCI_REMOTE_CI`
+ECO_VALIDATION_CONTAINER=`cat $CREDENTIALS_PATH/eco-validation-container/ECO_VALIDATION_CONTAINER`
+GITLAB_TOKEN=`cat $CREDENTIALS_PATH/gitlab-token/GITLAB_TOKEN`
+GITLAB_URL=`cat $CREDENTIALS_PATH/gitlab-url/GITLAB_URL`
+
+echo "Start"
+echo "SITE_NAME $SITE_NAME"
+echo "GITLAB_BRANCH $GITLAB_BRANCH"
+
+curl -X POST \
```

Comment:
This triggers the pipeline, we should also get the result of this so the job will fail if there are errors/failed tests